docs/pages/enroll-resources/application-access/okta/sync-scim.mdx

```diff
@@ -147,6 +147,53 @@ The synchronizer will create the following resources for each matched group or a
 It should be noted that the Access List sync waits until the Okta groups and Okta applications
 has finished syncing as Teleport resources, so it may not start synchronizing immediately on startup.
 
+### Handling Nested Access Lists
```

Comment:
```suggestion
### Handling nested Access Lists
```

We use sentence case in headings (this wasn't caught in `access-lists.mdx`, but it is in the [style guide](https://github.com/gravitational/docs/blob/main/docs-contributors/style-guide.md#headings)).